src/Sulu/Bundle/ContentBundle/Preview/Preview.php

```diff
@@ -175,20 +171,22 @@ private function update(
             );
         }
 
-        if ($render === true) {
+        try {
             $changes = $this->renderStructure($content, true, $property);
-            if ($changes !== false) {
-                $this->previewCache->appendChanges(
-                    [$property => $changes],
-                    $userId,
-                    $content->getUuid(),
-                    $webspaceKey,
-                    $locale
-                );
-            }
+        } catch (\Twig_Error $ex) {
+            // rendering error
+            $changes = [];
```

Comment:
Should we just ignore this? I think it should at least be logged, if not even show an error label in the administration. @chirimoya what do you think?